src/main/java/io/datanerds/avropatch/value/DefaultSchema.java

```diff
@@ -81,4 +81,11 @@
                     + "        \"logicalType\": \"uuid\"\n"
                     + "    }]\n"
                     + "}";
+    String TIMESTAMP =
```

Comment:
I still think this should be put into a resource file,  but I think I already said that...